extensions/3DTILES_metadata/1.0.0/README.md

```diff
@@ -0,0 +1,1623 @@
+      "byteOffset": 50528
+    },
+    {
+      "buffer": 0,
+      "byteLength": 8196,
+      "byteOffset": 75104
+    },
+    {
+      "buffer": 0,
+      "byteLength": 16388,
+      "byteOffset": 83304
+    }
+  ],
+  "tileAvailability": {
+    "bufferView": 0
+  },
+  "contentAvailability": {
+    "bufferView": 1
+  },
+  "childSubtreeAvailability": {
+    "bufferView": 2
+  },
+  "extensions": {
+    "3DTILES_metadata": {
+      "class": "tile",
+      "properties": {
+        "horizonOcclusionPoint": {
+          "bufferView": 3
+        },
+        "countries": {
+          "bufferView": 4,
+          "arrayOffsetBufferView": 5,
+          "stringOffsetBufferView": 6
+        }
+      }
+    }
+  }
+}
+```
+
+### Group Metadata
+
+Metadata may be assigned to groups. Groups represent collections of contents. Contents are assigned to groups with the `3DTILES_metadata` content extension.
+
+Group metadata can be paired with [`3DTILES_multiple_contents`](../../3DTILES_multiple_contents/0.0.0) to assign multiple contents in a tile to different groups.
+
+```jsonc
+{
+  "extensions": {
+    "3DTILES_metadata": {
+      "schema": {
+        "classes": {
+          "layer": {
+            "properties": {
+              "color": {
+                "type": "ARRAY",
+                "componentType": "UINT8",
+                "componentCount": 3
+              },
+              "priority": {
+                "type": "UINT32"
+              }
+            }
+          }
+        }
+      },
+      "groups": {
+        "buildings": {
+          "class": "layer",
+          "properties": {
+            "color": [128, 128, 128],
+            "priority": 0
+          }
+        },
+        "trees": {
+          "class": "layer",
+          "properties": {
+            "color": [10, 240, 30],
+            "priority": 1
+          }
+        }
+      }
+    }
+  },
+  "root": {
+    "refine": "ADD",
+    "geometricError": 32768.0,
+    "boundingVolume": {
+      "region": [-1.707, 0.543, -1.706, 0.544, -10.3, 253.113]
+    },
+    "children": [
+      {
+        "geometricError": 0.0,
+        "boundingVolume": {
+          "region": [-1.707, 0.543, -1.706, 0.544, -10.3, 253.113]
+        },
+        "content": {
+          "uri": "buildings.b3dm",
+          "extensions": {
+            "3DTILES_metadata": {
+              "group": "buildings"
+            }
+          }
+        }
+      },
+      {
+        "geometricError": 0.0,
+        "boundingVolume": {
+          "region": [-1.707, 0.543, -1.706, 0.544, -10.3, 253.113]
+        },
+        "content": {
+          "uri": "trees.i3dm",
+          "extensions": {
+            "3DTILES_metadata": {
+              "group": "trees"
+            }
+          }
+        }
+      }
+    ]
+  }
+}
+```
+
+### Feature Metadata
+
+A feature is an entity that has both geometry and metadata, such as a 3D building in a city. Features are the finest level of granularity within the tileset.
```

Comment:
Does this redefine **feature** compared to the 3D Tiles core spec?  There my intent was that a feature did not need to have metadata; in practice, it might have had metadata to allow for batching within a tile.